### ISSUE TITLE
`wiki_link` is in highlights.scm but doesn't appear to be compiled in?

### ISSUE THREAD

**Issue by SergioBenitez** [MDeiml/tree-sitter-markdown#118]
Hello!

I'm attempting to use this parser/grammar directly in a project. I'm getting an error when loading the `highlights` query due to an undeclared `wiki_links` node. It seems to me that the provided `highlights.scm` for the `inline` parser defines a capture for the node type, but the provided compiled `parser` and `scanner` don't actually define the node type. Am I understanding correctly?

If so, would it possible to have these two match, either by removing the `wiki_link` highlight or having the provided compiled artifacts be compiled with `wiki_link`?

Thank you!

**Comment by MDeiml** [comment]
You are correct. You can fix this either by removing the 'wiki_link` parts from your `.scm` file or by enabling the wiki-link extension.

Editing the `.scm` file is the easier solution. At the moment these extensions do not really play nicely with any editors. See the [README](https://github.com/MDeiml/tree-sitter-markdown#extensions) for more info, or #120 for how to maybe do this for neovim.
